apollo-router/src/plugins/response_cache/plugin.rs

```diff
@@ -618,6 +620,12 @@ impl CacheService {
         mut self,
         request: subgraph::Request,
     ) -> Result<subgraph::Response, BoxError> {
+        self.debug = self.debug
+            && (request
+                .supergraph_request
+                .headers()
+                .get(CACHE_DEBUG_HEADER_NAME)
+                == Some(&HeaderValue::from_static("true")));
```

Comment:
Nit: do we want to only accept the exact string `true` or should the presence of the header with any value be enough?

Reply:
For now I'm mainly consistent with how it works for the connectors debugger